test/e2e/network_segmentation.go

```diff
@@ -2453,7 +2453,7 @@ func runUDNPod(cs clientset.Interface, namespace string, serverPodConfig podConf
 			return v1.PodFailed
 		}
 		return updatedPod.Status.Phase
-	}, 2*time.Minute, 6*time.Second).Should(Equal(v1.PodRunning))
+	}, infraprovider.Get().GetPodReadyPollTimeout(), infraprovider.Get().GetPodReadyPollInterval()).Should(Equal(v1.PodRunning))
```

Comment:
I would define this as deployment config actually.

I would refactor this to use as much as possible helpers already available in framework pod package: WaitTimeoutForPodRunningInNamespace, WaitTimeoutForPodRunningReadyInNamespace, ...

I don't think we need to define an interval, if we use the framework helpers it will default to TestContext.timeouts.Poll which is 2s, that should be good enough?

It might be usefull to align our timeouts to framework ones:
```
var defaultTimeouts = TimeoutContext{
	Poll:                      2 * time.Second, // from the former e2e/framework/pod poll interval
	PodStart:                  5 * time.Minute,
	PodStartShort:             2 * time.Minute,
	PodStartSlow:              15 * time.Minute,
```